tests/jvm/src/test/resources/metacp.expect

```diff
@@ -1594,10 +1594,10 @@ _root_.types.Test.C#compoundType1. => private[this] val field compoundType1: Any
   AnyRef => _root_.scala.AnyRef#
   k => _root_.types.Test.C#`<refinement>`#k().
   Int => _root_.scala.Int#
-_root_.types.Test.C#compoundType2(). => val method compoundType2: : M with N
+_root_.types.Test.C#compoundType2(). => val method compoundType2: : M with N {  }
```

Comment:
Would be nice to skip these braces if decls.isEmpty

Reply:
There's no way to distinguish `M with N` from `M with N {}`, so let's do the easiest possible thing, i.e. nothing at all. There are clauses in the spec that explicitly mark areas where we don't guarantee 100% correspondence between syntax and SemanticDB, and this is one of them.